.travis.yml

```diff
@@ -51,3 +51,13 @@ script:
 
 after_success:
   - if [[ "$WITH_COVERAGE" == "true" ]]; then php composer/bin/test-reporter; fi
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
That's even better, I personally prefer to provide keys in `.travis.yml` because they can't be deleted without a code change.

It's a lot easier to accidentally delete a key from https://travis-ci.org/codeclimate/php-test-reporter/settings, hehe!